WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Nothing shall be blocked, nothing shall be thanked. You can explain elsewhere your desired changes and they will evolve from that.

I've explained my desired changes to you over and over again. It's not my fault if you don't have the means to understand, by now, what I want to do. You've already seen the code in the Mercenaries file, and you have no legitimate reasons for not wanting those changes to be accepted.

Just because you don't agree with something, doesn't mean it shouldn't be allowed to go into the Mod. There are things in this Mod that I absolutely disagree with, but fighting them is futile and I let it slide. Maybe you should get off your high horse and do the same for once in your Modding career, and merge this PR!

I can go elsewhere and get someone to do it if you aren't willing to listen to me.